components/threading/channel_threads/thread_footer/__snapshots__/thread_footer.test.tsx.snap

```diff
@@ -989,7 +989,7 @@ exports[`components/threading/channel_threads/thread_footer should report total
               values={
                 Object {
                   "formatted": <Memo(SemanticTime)
-                    aria-label="Mon Apr 01 2019 23:31:44 GMT+0000"
+                    aria-label="Mon Apr 01 2019 19:31:44 GMT-0400"
```

Comment:
These updates seem machine dependent. I wonder if there's some sort of config that needs to be passed so that they only get rendered on UTC time code. I'm guessing we'll want to revert these. We'll see what CI thinks of them.